pages/linux/cloud-init.md

```diff
@@ -0,0 +1,24 @@
+# cloud-init
+
+> Command line tool for managing cloud instance initialization.
+> More information: <https://cloudinit.readthedocs.io>.
+
+- Display the status of the most recent cloud-init run:
+
+`cloud-init status`
+
+- Wait for cloud-init to finish running, then report status:
```

Comment:
Although it is fine as is; how about an `and` instead of `,`:
```suggestion
- Wait for cloud-init to finish running and then report status:
```

Reply:
I'll apply this because is minor.

Reply:
Thank you both